src/styles/card.scss

```diff
@@ -0,0 +1,48 @@
+.card {
```

Comment:
why don't you use parent selector?
https://sass-lang.com/documentation/style-rules/parent-selector